lib/node_modules/@stdlib/math/base/ops/imuldw/benchmark/benchmark.js

```diff
@@ -49,3 +49,27 @@ bench( pkg, function benchmark( b ) {
 	b.pass( 'benchmark finished' );
```

Comment:
We should change L41 to be `y[ 0 ]` and L46 to be `y[ 1 ]`, since this API no longer returns `NaN`.